app/routes/application.vue

```diff
@@ -0,0 +1,61 @@
+<template lang="html">
+  <div class="">
```

Comment:
Wrap this in a `.section` and a `.container` to get the double container working.